dangerfile.js

```diff
@@ -0,0 +1,92 @@
+const {danger, fail, warn} = require('danger');
+const yaml = require('js-yaml');
+const Ajv = require('ajv');
+const {readFileSync} = require('fs');
+
+// JSON Schema for changelog entries.
+// @see https://json-schema.org/
+const CHANGELOG_SCHEMA = {
+  "definitions": {
+    "ChangelogLine": {
+      "project": "object",
```

Comment:
PR # is something we want to enforce on regular branches, except for dependentbot branches. 
I think you can use a library to get the current branch and if it's not dependentbot branches e.g. `!branchName.match(/^(?!dependbot).*/g)` add "pr" to required list, and break #PR into its own field

Reply:
node git libraries: 
https://www.npmjs.com/package/current-git-branch 
https://www.npmjs.com/package/simple-git

Reply:
Another way I can think of is that we pull out the circle steps into its own job, and then add to the workflow and exclude the dependabot branch, and also master and develop branches. I will need to update the dependabot auto changelog script either way to match the new format.